modules/fbs/prancer/SwaggerApiError.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Reload\Prancer;
```

Comment:
This is for sure outside this PR's scope, but should the namespace have been DDB and not Reload?